middleman_protocol/middleman_protocol/tests/test_stream.py

```diff
@@ -392,3 +392,20 @@ def test_that_raising_error_in_generator_should_call_close_on_socket(self, unuse
                             next(split_stream_generator)
 
                 mock_socket_close.assert_called_once()
+
+    def test_that_when_socket_receives_no_bytes_socket_error_is_raised(self, unused_tcp_port):
```

Comment:
************* Module middleman_protocol.tests.test_stream
R:396, 4: Method could be a function (no-self-use)